core/src/main/scripts/importer/importOncokbDiscreteCNA.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2020 The Hyve B.V.
+# This code is licensed under the GNU Affero General Public License (AGPL),
+# version 3, or (at your option) any later version.
+#
+
+#
+# This file is part of cBioPortal.
+#
+# cBioPortal is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""OncoKB annotation import script for discrete CNA files.
+"""
+
+import argparse
+import importlib
+import logging.handlers
+import os
+import requests
+import sys
+from os import path
+from pathlib import Path
+
+# configure relative imports if running as a script; see PEP 366
+# it might passed as empty string by certain tooling to mark a top level module
+if __name__ == "__main__" and (__package__ is None or __package__ == ''):
+    # replace the script's location in the Python search path by the main
+    # scripts/ folder, above it, so that the importer package folder is in
+    # scope and *not* directly in sys.path; see PEP 395
+    sys.path[0] = str(Path(sys.path[0]).resolve().parent)
+    __package__ = 'importer'
+    # explicitly import the package, which is needed on CPython 3.4 because it
+    # doesn't include https://github.com/python/cpython/pull/2639
+    importlib.import_module(__package__)
+
+from . import cbioportal_common
+from . import libImportOncokb
+from . import validateData
```

Comment:
Sorry about this silly question, is it by design having these three imports written after the logics above? Instead of putting all imports together?

Reply:
I guess the logic above is necessary because we are not installing the python package but running it as a script. The logic above enables the relative imports listed here

Reply:
Indeed